README.md

```diff
@@ -85,7 +91,7 @@ Our general [development guide](http://opensphere.readthedocs.io/en/latest/getti
 
 ## Contributing
 
-To get involved with OpenSphere directly, see our [contributing guide](https://github.com/ngageoint/opensphere/blob/master/CONTRIBUTING.md).
+To get involved with OpenSphere directly, see our [contributing guide](http://localhost:8080/workspace/os/workspace/opensphere/docs/_build/html/contributing.html).
```

Comment:
I think the github link is better. The local path is potentially variable depending on how your yarn is exposed.